bucket/r-devel.json

```diff
@@ -0,0 +1,84 @@
+{
+    "homepage": "https://www.r-project.org",
+    "description": "A free software environment for statistical computing and graphics.",
+    "version": "devel",
+    "license": "GPL-2.0-only",
+    "url": "https://cran.r-project.org/bin/windows/base/R-devel-win.exe",
+    "hash": "183a69e53b35df320a24915af2181b7ebb9f0a335a33f641fcd40d1ed8af21cb",
+    "innosetup": true,
+    "architecture": {
+        "64bit": {
+            "pre_install": "Rename-Item \"$dir\\bin\\R,2.exe\" 'R.exe'",
+            "bin": [
+                "bin\\x64\\R.exe",
+                "bin\\x64\\Rcmd.exe",
+                "bin\\x64\\Rgui.exe",
+                "bin\\x64\\Rscript.exe",
+                "bin\\x64\\Rterm.exe"
+            ],
+            "shortcuts": [
+                [
+                    "bin\\x64\\Rgui.exe",
+                    "R"
+                ]
+            ]
+        },
+        "32bit": {
+            "pre_install": "Rename-Item \"$dir\\bin\\R,1.exe\" 'R.exe'",
+            "bin": [
+                "bin\\i386\\R.exe",
+                "bin\\i386\\Rcmd.exe",
+                "bin\\i386\\Rgui.exe",
+                "bin\\i386\\Rscript.exe",
+                "bin\\i386\\Rterm.exe"
+            ],
+            "shortcuts": [
+                [
+                    "bin\\i386\\Rgui.exe",
+                    "R"
+                ]
+            ]
+        }
+    },
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\bin\\R,*.exe\" -Force",
+            "Copy-Item \"$dir\\bin\\R.exe\" \"$dir\\bin\\Rscript.exe\"",
+            "if (Test-Path (appdir 'rtools' $global)) {",
+            "   $renvloc = \"$dir\\etc\\Renviron.site\"",
+            "   $rtoolsloc = versiondir 'rtools' (current_version 'rtools' $global) $global",
+            "   if (Test-Path $renvloc) {",
+            "       $renv = Get-Content $renvloc -Encoding ASCII",
+            "   } else {",
+            "       $renv = @()",
+            "   }",
+            "   $rtoolspath = \"$rtoolsloc\\usr\\bin\"",
+            "   if ($renv -cmatch 'PATH=') {",
+            "       $renv = $renv -replace 'PATH=\"(.*)', ('PATH=\"' + $rtoolspath + ';$1')",
+            "   } else {",
+            "       $renv += \"`r`nPATH=\", '\"', $rtoolspath, ';${PATH}\"' -join ''",
+            "   }",
+            "   $renv | Set-Content $renvloc -Encoding ASCII",
+            "}"
```

Comment:
@andrie what about removing all this part ? to avoid issues. 

I don't think we need this app to modify .Renviron.

One can modify on its own the .Renviron.